src/main/java/frc/robot/controls/Driver.java

```diff
@@ -2,20 +2,38 @@
 
 import edu.wpi.first.wpilibj.Joystick;
 import frc.robot.Constants;
+import java.lang.Math;
 
 public class Driver
 {
     public static Joystick throttle = new Joystick(Constants.THROTTLE_JOYSTICK);
     public static Joystick steer = new Joystick(Constants.STEER_JOYSTICK);
 
+    private static double throttleVal = 0;
+    private static double steerVal = 0;
+
     public static double getThrottle()
     {
-        return -throttle.getY();
+        throttleVal = -throttle.getY();
+
+        if (Math.abs(throttleVal) < Constants.DEADZONE)
+        {
+            throttleVal = 0;
+        }
+
+        return throttleVal;
```

Comment:
You don't really need a variable here.  If < DEADZONE, just return 0, else return throttle.getY()

Reply:
Actually I take it back.  The stick value could change between checking the ABS and returning.  It is correct to store it in a variable.

Reply:
I would just say return 0 instead of saving it to `throttleVal` (save a write to RAM/cache)